### PR TITLE
DOCS-1508: Clarify fleet use case

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by npentrel — CHANGES_REQUESTED** [review]
I think you can go further than that: 
Start in the first sentence "You can use Viam’s cloud-based fleet management tools to monitor and manage access to your fleet of smart machines." you can make that about an existing fleet. 

In the second paragraph and the step where you configure robots, add info that you can add management and this regardless of whether the machines are already running code. that this works alongside it. And that you can once you've added viam, use it for any of or all of data management, remote control, and code deployment. (Probably not in exactly those words ;))

**Comment by JessamyT** [comment]
Between step 5 talking about data and the control tab, plus the registry sentence, I think the last point is addressed?

File: docs/use-cases/manage-fleet.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ images: ["/fleet/locations.png"]
 ---
 
 You can use Viam's cloud-based fleet management tools to monitor and manage access to your fleet of {{< glossary_tooltip term_id="smart-machine" text="smart machines" >}}.
+Use these tools as you create and scale a new fleet of robots, or [integrate Viam](https://www.viam.com/post/integrate-your-ros2-robot-with-viam) to manage and add functionality to your existing fleet.
```
Comment (npentrel) [inline docs/use-cases/manage-fleet.md:15]: This is a good idea but I think remove the link here and add it to the next steps section instead. you can remove the tutorial one to make space

Comment (npentrel) [inline docs/use-cases/manage-fleet.md:15]: Nit - I think that helps understanding
```suggestion
Use these tools as you create and scale a new fleet of robots, or [integrate Viam](https://www.viam.com/post/integrate-your-ros2-robot-with-viam) to manage and add functionality like data management to your existing fleet.
```

**Review by npentrel — APPROVED** [review]
This looks good to me - would you mind adding Eliot to this review, just in case he wants to chime in?

File: docs/use-cases/manage-fleet.md [hunk 2]
```diff
@@ -56,5 +57,11 @@ You can grant users different levels of access to individual robots or to groups
 {{% card link="/fleet/" %}}
 {{% card link="/fleet/cli/" %}}
 {{% card link="/fleet/rbac/" %}}
-{{% card link="/tutorials/" %}}
+{{% manualcard link="https://www.viam.com/post/integrate-your-ros2-robot-with-viam/" target="_blank" %}}
```
Comment (npentrel) [inline docs/use-cases/manage-fleet.md:60]:
```suggestion

<!-- markdownlint-disable MD034 -->

{{% manualcard link="https://www.viam.com/post/integrate-your-ros2-robot-with-viam/" target="_blank" %}}
```

**Comment by npentrel** [comment]
Let's merge this and we can address any feedback from Eliot later. Just fixing the test

**Pull request merged**